src/legacy/core_plugins/kibana/public/home/plugin.ts

```diff
@@ -110,6 +111,7 @@ export class HomePlugin implements Plugin {
 
   start(core: CoreStart, { data, home }: HomePluginStartDependencies) {
     this.environment = home.environment.get();
+    this.featureCatalogue = home.featureCatalogue;
```

Comment:
Can't get `featureCatalogue` right here, because of registering features in legacy `management` plugin